substratevm/src/com.oracle.svm.core/src/com/oracle/svm/core/jdk/LocalizationSupport.java

```diff
@@ -95,12 +96,13 @@ void includeResourceBundles() {
      *
      * @param locale this parameter is not currently used.
      */
-    public ResourceBundle getCached(String baseName, Locale locale) {
+    public ResourceBundle getCached(String baseName, Locale locale) throws MissingResourceException {
         ResourceBundle result = cache.get(baseName);
         if (result == null) {
             String errorMessage = "Resource bundle not found " + baseName + ". " +
                             "Register the resource bundle using the option " + includeResourceBundlesOption + baseName + ".";
-            throw VMError.unsupportedFeature(errorMessage);
+            System.err.println(errorMessage);
```

Comment:
Is the `System.err.println` needed given that you throw a `MissingResourceException` below?